src/20_Components/amp-reddit.html

```diff
@@ -0,0 +1,65 @@
+<!--
+  ## Introduction
+
+  The [amp-reddit](https://www.ampproject.org/docs/reference/components/amp-reddit) component allows embedding both reddit posts and comments into AMP files.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8">
+  <title>amp-reddit</title>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <!-- ## Setup -->
+    <!-- Import the amp-reddit component -->
+    <script async custom-element="amp-reddit" src="https://cdn.ampproject.org/v0/amp-reddit-0.1.js"></script>
+  <link rel="canonical" href="<%host%>/components/amp-reddit/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  </head>
+  <body>
+
+    <!-- ## Basic Usage -->
+    <!-- The `amp-reddit` component requires to be told that the embed is a post/comment as well as the source of the embed -->
+    <!-- ### Embedding a post -->
+    <amp-reddit
+      layout="responsive"
+      width="300"
+      height="400"
+      data-embedtype="post"
+      data-src="https://www.reddit.com/r/me_irl/comments/52rmir/me_irl/?ref=share&amp;ref_source=embed">
+    </amp-reddit>
+
+    <!-- ### Embedding comments -->
+    <amp-reddit
+      layout="responsive"
+      width="400"
+      height="400"
+      data-embedtype="comment"
+      data-src="https://www.reddit.com/r/me_irl/comments/52rmir/me_irl/d7n3soo/">
+    </amp-reddit>
+
+  <!-- ## Customization for embedding comments -->
+
+  <!-- ### Embed the parent comment. -->
```

Comment:
can you shorten the title (and remove the dot at the end) and add a description, e.g.:

```
<!-- 
### Parent Comment

Embed the parent comment by specifying `data-embedparent="true"`.

-->
```

Reply:
Done.